shared/actions/kbfs/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
```

Comment:
so i think this file needs to be slightly different. the way the packaging works if you try and resolve a file it'll look for the platform specific stuff first, then fallback. so it'll try
index.ios.js then index.native.js then index.js (for desktop it'll be index.desktop.js then index.js) so if the intention is for fsList to work everywhere (which it should) but have openInKBFS work differently then we should have index.js have fsList and move openInKBFS into something like platform.native.js and platform.desktop.js, then have this openInKBFS delegate to the one from platform , so like

this file
```
import {openInKBFS as platformOpenInKBFS} from './platform'

export function openInKBFS (path: string = ''): AsyncAction {
  return platformOpenInKBFS(path)
}
```